files/playbooks/octopus/ceph-bootstrap-dashboard.yml

```diff
@@ -36,7 +36,7 @@
       no_log: true
 
     - name: Create admin user
-      ansible.builtin.command: "ceph dashboard ac-user-create {{ ceph_dashboard_username }} administrator -i /data/ceph_dashboard_password"  # noqa 301
+      ansible.builtin.command: "ceph dashboard ac-user-create {{ ceph_dashboard_username }} administrator -i /data/ceph_dashboard_password"  # noqa no-changed-when
```

Comment:
This one shouldn't be necessary, because in fact there is a `changed-when` term below.